apps/docs/src/pages/about.tsx

```diff
@@ -0,0 +1,264 @@
+import {
+  faArrowLeft,
+  faCheck,
+  faCrown,
+  faPlus,
+} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import Layout from '@theme/Layout'
+import TextTransition, { presets } from 'react-text-transition'
+
+const people = [
+  {
+    name: 'Roland Schläfli',
+    imageUrl:
+      'https://www.df.uzh.ch/contacts/df/admin/teaching-center/rschl%C3%A4fli/photo/Schl%C3%A4fli-Roland.jpg.jpg',
+    githubUrl: 'https://github.com/rschlaefli',
```

Comment:
It would definitely be good add an e-mail address here and/or the link to the personal contact information at the department (e.g. https://www.df.uzh.ch/en/people/staff/teaching-center/roland-schlaefli.html)